services/brig/src/Brig/IO/Intra.hs

```diff
@@ -191,25 +194,37 @@ push :: List1 Event  -- ^ The events to push.
      -> Push.Route   -- ^ The push routing strategy.
      -> Maybe ConnId -- ^ The originating device connection.
      -> AppIO ()
```

Comment:
No logic changes here, only the split of `push` into `push` and `rawPush`. The reason I need `rawPush` is that `push` only allows pushing Brig events, but here I want to push a Galley event.